flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -262,12 +262,13 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
             }, this.getDelaySwitchPage());
         };
 
-        CDUMenuPage.ShowPage(this);
+        //CDUMenuPage.ShowPage(this);
```

Comment:
In general, we don't keep around comments of old code like this. We can use version control if we ever want to recover a previous state.